fhir-persistence/src/test/java/com/ibm/fhir/persistence/search/test/AbstractSearchDateTest.java

```diff
@@ -631,11 +635,17 @@ public void testSearchDate_date_missing() throws Exception {
     }
     @Test
     public void testSearchDate_date_chained() throws Exception {
-        // Date is specific - 2018-10-29
-        assertSearchReturnsComposition("subject:Basic.date", "2018-10-29");
-        assertSearchDoesntReturnComposition("subject:Basic.date", "2018-10-29T17:12:00-04:00");
-        assertSearchDoesntReturnComposition("subject:Basic.date", "2018-10-29T17:12:00");
-        assertSearchDoesntReturnComposition("subject:Basic.date", "2025-10-29");
+        final String METHOD = "testSearchDate_date_chained";
+        logger.entering(CLASSNAME, METHOD);
```

Comment:
Best practice with test-ng is to use @beforetest and @aftertest.  it may be worth parking that in your arsernal

Reply:
How does this relate to the highlighted lines?